src/Runner.Worker/ExecutionContext.cs

```diff
@@ -673,6 +673,11 @@ public void InitializeJob(Pipelines.AgentJobRequestMessage message, Cancellation
                 Environment.SetEnvironmentVariable(Constants.Variables.Agent.ForcedNodeVersion, "node12");
             }
 
+            if (Global.Variables.GetBoolean("DistributedTask.ForceGithubJavascriptActionsToNode16") ?? false)
+            {
+                Environment.SetEnvironmentVariable(Constants.Variables.Agent.ForcedActionsNodeVersion, "node16");
```

Comment:
I would suggest we don't use an environmental variable to control this, since we can fully control the switch via feature flag from the service side.